assets/javascript/script.js

```diff
@@ -0,0 +1,70 @@
+const appearMobileMenu = document.querySelector('.mobile-menu');
+const disAppearMobileMenu = document.querySelector('.mobile-menu');
+
+function mobileMenu() {
+  const appearMobileDivision = document.createElement('div');
+  appearMobileDivision.classList.add('mobile-menu-div');
+  appearMobileDivision.setAttribute('id', 'rmv');
+  appearMobileMenu.appendChild(appearMobileDivision);
+
+  const appearMobileDiv = document.createElement('div');
+  appearMobileDiv.classList.add('mobile-menu-close');
+  appearMobileDivision.appendChild(appearMobileDiv);
+
+  const appearMobileImg = document.createElement('img');
+  appearMobileImg.classList.add('mobile-menu-close-icon');
+  appearMobileDiv.appendChild(appearMobileImg);
+  appearMobileImg.setAttribute('src', 'assets/images/close-icon.svg');
+  appearMobileImg.setAttribute('onclick', 'disappearMobileMenu()');
+  appearMobileImg.setAttribute('id', 'disappear');
+
+  const appearMobileUl = document.createElement('ul');
+  appearMobileUl.classList.add('mobile-list');
+  appearMobileDivision.appendChild(appearMobileUl);
+
+  const appearMobileUlLi1 = document.createElement('li');
+  appearMobileUlLi1.classList.add('mobile-list-item');
+  appearMobileUl.appendChild(appearMobileUlLi1);
+
+  const appearMobileLink1 = document.createElement('a');
+  appearMobileLink1.classList.add('mobile-link');
+  appearMobileUlLi1.appendChild(appearMobileLink1);
+  appearMobileLink1.setAttribute('href', '#Portfolio');
+  appearMobileLink1.textContent = 'Portfolio';
+
+  const appearMobileUlLi2 = document.createElement('li');
+  appearMobileUlLi2.classList.add('mobile-list-item');
+  appearMobileUl.appendChild(appearMobileUlLi2);
+
+  const appearMobileLink2 = document.createElement('a');
+  appearMobileLink2.classList.add('mobile-link');
+  appearMobileUlLi2.appendChild(appearMobileLink2);
+  appearMobileLink2.setAttribute('href', '#About');
+  appearMobileLink2.textContent = 'About';
+
+  const appearMobileUlLi3 = document.createElement('li');
+  appearMobileUlLi3.classList.add('mobile-list-item');
+  appearMobileUl.appendChild(appearMobileUlLi3);
+
+  const appearMobileLink3 = document.createElement('a');
+  appearMobileLink3.classList.add('mobile-link');
+  appearMobileUlLi3.appendChild(appearMobileLink3);
+  appearMobileLink3.setAttribute('href', '#Contact');
+  appearMobileLink3.textContent = 'Contact';
```

Comment:
- [ ] Each of the links should take the user to the specific part of the page when clicked. 

![image](https://user-images.githubusercontent.com/81932478/189080767-e3feb4e7-6ee8-4acd-bafc-33a7be01c8e1.png)

- The Portfolio link in the image should take the user to the `Portfolio` part of the web page
- The About link in the image should also take the user to the `About` part of the web page.
- The Contact link in the mobile menu should also take the user to the `Contact`, part of the page.